target/linux/ath79/patches-4.19/0028-MIPS-ath79-drop-machfiles.patch

```diff
@@ -42,7 +42,7 @@ Signed-off-by: John Crispin <john@phrozen.org>
 --- a/arch/mips/ath79/Kconfig
 +++ b/arch/mips/ath79/Kconfig
 @@ -1,79 +1,6 @@
- # SPDX-License-Identifier: GPL-2.0
+ # SPDX-License-Identifier: GPL-2.0-only
```

Comment:
This line is from kernel, we don't add/remove it, but just use it to determine the position of the changes. Lines like this must not be changed.

Reply:
And personally I'm not sure whether we should touch the patches at all.

Reply:
I removed the target patch for now.